src/screens/Main/components/Card/Card.tsx

```diff
@@ -57,9 +57,9 @@ const styles = StyleSheet.create({
     color: '#00A8F3',
   },
 });
-export const Card = ({product}: TCardProps): JSX.Element => {
+export const Card = ({product, onPress}: TCardProps): JSX.Element => {
   return (
-    <View style={styles.container}>
+    <Pressable style={styles.container} onPressIn={() => onPress(product.id)}>
```

Comment:
I would say that pressIn is too early to react on any action performed by the user